### PR TITLE
1.10.15

### PR DESCRIPTION
**Pull request by krzychu124**
### Changelog
1.10.15, 10/02/2019
- Enhancement: Now you can use Escape key to close Traffic Manager without returning to Pause Menu (thanks to @aubergine10 for suggestion) (#16)
- Gameplay: Updated pathfinding with missing vanilla logic
- Gameplay: Tweaked values in CargoTruckAI path finding (thanks to @pcfantasy for improvement suggestion)
- Gameplay: Tweaked speed multiplier of reckless drivers to get more realistic speed range (thanks to @aubergine10 for suggestion) (#23)
- UI: New icons for cargo and passenger train restriction (thanks to @aubergine10) (#17)
- Translations: Simplified Chinese translation updated (thanks to @Emphasia for translating)
- Other: Added notification if user is still subscribed to old original TM:PE
- [Experimental feature] Turn on red (thanks to @FireController1847 for implementation and to @pcfantasy for source code base)

### CODE REVIEW

**Comment by krzychu124** [comment]
Any questions? 😄

**Comment by FireController1847** [comment]
Just so you're aware, I was having issues with the escape button thing, don't know if we should save that for 1.10.16. Sorry I didn't report it earlier, I forgot >.<

**Comment by aubergine10** [comment]
Wasn't there a language file update from someone?

**Comment by krzychu124** [comment]
Yup I know about this behavior and it seems that I can't fix that without overwriting core menu panel (Left sidebar)

**Comment by krzychu124** [comment]
With regards to Esc key it's easy to reproduce, just open any TMPE tool then select any other in-game overlay from left sidebar (electricity, pollution whatever) and now hit Esc... 
Everything will close because sidebar menu panel is focused and its default action on Esc key pressed is close everything (and cancel event propagation after).
I will try to improve it later today.

**Comment by pcfantasy** [comment]
Nice work！

File: TLM/TLM/Resources/lang_zh.txt [hunk 1]
```diff
@@ -192,6 +194,7 @@ Window_transparency 窗口透明度
 Overlay_transparency 覆盖透明度
 Show_error_message_if_a_mod_incompatibility_is_detected 检测到MOD不兼容时提示
 Node_is_level_crossing 此节点级别交汇，不可禁用红绿灯。
+Experimental_features [Experimental features]:
```
Comment (Emphasia) [inline TLM/TLM/Resources/lang_zh.txt]: Experimental_features [实验特性]:

File: TLM/TLM/Resources/lang_zh.txt [hunk 2]
```diff
@@ -58,6 +58,8 @@ Enable_Advanced_Vehicle_AI 启用高级车辆AI
 Vehicles_may_enter_blocked_junctions 车辆可以进入堵塞路口加塞
 All_vehicles_may_ignore_lane_arrows 所有车辆可忽略车道方向
 Busses_may_ignore_lane_arrows 公交车可忽略车道方向
+Turn_on_red_enabled_by_default 默認情況下啟用交叉點處的紅色啟用
+Turn_on_red 车辆可能会在红色交通灯处转弯
```
Comment (Emphasia) [inline TLM/TLM/Resources/lang_zh.txt]: Turn_on_red 车辆红灯可转弯

File: TLM/TLM/Resources/lang_zh.txt [hunk 3]
```diff
@@ -58,6 +58,8 @@ Enable_Advanced_Vehicle_AI 启用高级车辆AI
 Vehicles_may_enter_blocked_junctions 车辆可以进入堵塞路口加塞
 All_vehicles_may_ignore_lane_arrows 所有车辆可忽略车道方向
 Busses_may_ignore_lane_arrows 公交车可忽略车道方向
+Turn_on_red_enabled_by_default 默認情況下啟用交叉點處的紅色啟用
```
Comment (Emphasia) [inline TLM/TLM/Resources/lang_zh.txt]: Turn_on_red_enabled_by_default 默认启用「红灯可转弯」

**Pull request merged**